admin_manual/installation/deployment_recommendations.rst

```diff
@@ -26,6 +26,15 @@ General Recommendations
 .. note:: Whatever the size of your organization, always keep one thing in mind: 
    *The amount of data stored in ownCloud will only grow. So plan ahead.*
 
+ownCloud Administrators Must Have Command Line or Cron Access
+-------------------------------------------------------------
+
+We only recommend using hosts that provide command-line or Cron access (ideally both) *to ownCloud administrator*, for three key reasons:
+
+#. Without command-line access, :doc:`OCC commands <../configuration/server/occ_command>`, required for administrative tasks such as repairs and upgrades, are not available.
+#. Without crontab access, you cannot run background jobs reliably. :ref:`ajax/cron.php <ajax_cron_label>` is available, but it is not reliable enough and is only available when using the web UI. Additionally, ownCloud relies heavily on :ref:`background jobs <available_background_jobs_label>` especially for long-running operations, which will likely cause PHP timeouts.
```

Comment:
=> "not reliable enough" might be enough. Else if you want to explain then expand to "not reliable enough because it only runs when people are using the web UI, not when using only desktop or mobile clients"

Reply:
like @PVince81 says, the reason it is "not reliable enough" is just that if everybody just uses sync clients and not the web UI, then it never gets invoked. Maybe:
s/and is only available when using the web UI/because it only runs when people are using the web UI